README.md

```diff
@@ -55,6 +65,7 @@ options:
   -v, --version         show program's version number and exit
   -r CRAM_REFERENCE, --cram-reference CRAM_REFERENCE
                         Path to reference file for CRAM
+  -c CPUS, --cpus CPUS  Number of CPUs to parallelize over when validating multiple files
```

Comment:
Minor point - "CPU" is a misnomer for this change. There is no guarantee regarding the number of CPUs, cores, threads, etc. used. Consider renaming this to represent that this only changes how many _processes_ are started.

Reply:
Fair point, I changed the argument to `processes` instead